sycl/plugins/unified_runtime/ur/adapters/level_zero/context.cpp

```diff
@@ -331,7 +331,8 @@ ur_result_t ContextReleaseHelper(ur_context_handle_t Context) {
   // Technically it should be placed to the destructor of ur_context_handle_t
   // but this makes API error handling more complex.
   if (DestroyZeContext) {
-    auto ZeResult = ZE_CALL_NOCHECK(zeContextDestroy, (DestroyZeContext));
+    ze_result_t ZeResult{};
+    ZE_CALL(zeContextDestroy, (DestroyZeContext), ZeResult);
```

Comment:
What the purpose if such rewrite?

Reply:
ZE_CALL_NOCHECK in this line needs to use ZeResult, but in order to be able to print "ZE --->" before the execution of the call, I have to execute a urPrint before, so now I have two instructions to execute in the macro, instead of just one, as it was the original macro. 

By passing ZeResult as a parameter in the macro, we can still return ZeResult and be able to print "ZE --->" before the execution of the ze call.

Reply:
I see, still wanted to see NOCHECK on macro name, I think